source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
+                        <p>Some printed scholarly editions like the <hi rend="italic">Neue Bach-Ausgabe</hi> provide very detailed information about the sizes and binding of individual leaves of a manuscript; with <gi scheme="MEI">foliaDesc</gi> and its children it is possible to encode that information in processable ways, even without providing digitizations of the sources via <gi scheme="MEI">surface</gi>.</p>
+                     </div>
+                     <div xml:id="patches" type="div4">
+                        <head>Patches</head>
+                        <p>Sometimes, manuscripts (but also prints) are subject to modifications that do not change the textual content, but the actual physical item. Typical examples for this are patches glued on a page, or cutouts. Both these situations can be encoded inside <gi scheme="MEI">foliaDesc</gi>.</p>
+                        <p>A patch is an additional writing surface attached to one of the sides of a <gi scheme="MEI">folium</gi> or <gi scheme="MEI">bifolium</gi>:</p>
+                        <p>
+                           <specList>
+                              <specDesc key="patch"/>
+                           </specList>
+                        </p>
+                        <p>The <gi scheme="MEI">patch</gi> element is placed inside the <gi scheme="MEI">folium</gi> or <gi scheme="MEI">bifolium</gi> to which it is attached. To which side of this parent it is attached is specified using the (required) <att>attached.to</att> attribute:</p>
+                        <p>
+                           <specList>
+                              <specDesc key="patch" atts="attached.to"/>
+                           </specList>
+                        </p>
+                        <p>Depending on the parent, allowed values for <att>attached.to</att> are either "<hi rend="italic">recto</hi>" and "<hi rend="italic">verso</hi>" (in case of <gi scheme="MEI">folium</gi>) or "<hi rend="italic">outer.recto</hi>", "<hi rend="italic">inner.verso</hi>", "<hi rend="italic">inner.recto</hi>" and "<hi rend="italic">outer.verso</hi>" (in case of <gi scheme="MEI">bifolium</gi>).</p>
+                        <p>The exact position of the patch on the underlying surface may be specified using the optional <att>x</att> and <att>y</att> attributes, which are used to specify the distance from the upper left corner of the patch from the upper left corner of the surface it is attached to. At this point, it is not possible to specify rotation.</p>
```

Comment:
We should probably specify what units are used for `x` and `y`, no?

Reply:
Added to issue: https://github.com/music-encoding/metadata-ig/issues/25